arithmetization/src/main/java/net/consensys/linea/bl/TransactionProfitabilityCalculator.java

```diff
@@ -25,68 +25,58 @@
 
 @Slf4j
 public class TransactionProfitabilityCalculator {
-
+  private static final long TO_WEI_MULTIPLIER = 1_000;
```

Comment:
I think "Wei in kWei" makes more sense. Even though it's used with `variableCostKWei` and `variableCostKWei` meaning of this constant itself is a little implicit. And in case of a change of any of the costs from kwei to mwei, for example, we could catch a bug

Reply:
Renamed, indeed I was not completely satisfied with my choice